test/data/notificationDestinationTransferPrepared.json

```diff
@@ -0,0 +1,19 @@
+{
+  "id": "http://eur-ledger.example/transfers/3968e526-b5ef-4c35-8923-0e7cfd0892d3",
+  "ledger": "http://eur-ledger.example",
+  "debits":[{
+    "amount":"1.00",
+    "account":"http://eur-ledger.example/accounts/mark",
+    "memo": {
+      "source_transfer_ledger": "http://usd-ledger.example",
+      "source_transfer_id": "6851929f-5a91-4d02-b9f4-4ae6b7f1768c"
+    }
+  }],
+  "credits":[{
+    "amount":"1.00",
+    "account":"http://eur-ledger.example/accounts/bob"
+  }],
+  "execution_condition": "cc:0:3:vmvf6B7EpFalN6RGDx9F4f4z0wtOIgsIdCmbgv06ceI:7",
+  "expires_at": "2015-06-16T00:00:02.000Z",
+  "state": "proposed"
```

Comment:
state probably should be `prepared` for a `notificationDestinationTransferPrepared`